src/resources/Sources/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './Sources';
 export * from './SourcesInterfaces';
 export * from './SourcesFields/';

```

Comment:
Non-bidding either, but it'd be nice to uniformize here and make it goes as such: 
```suggestion
export * from './SourcesFields';
```

Just to be consistent.